integrations/astro/agnostic-astro/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@astrojs/agnostic-astro",
```

Comment:
I feel like I may be violating an internal integrations naming scheme here with the @astrojs/ part (although I would love to convert this to an internally blessed integration if it was "in the cards" 😛 )